src/combine.jl

```diff
@@ -21,12 +19,33 @@ function merge{T}(ta1::TimeArray{T}, ta2::TimeArray{T}; col_names=[""])
     # combine existing colnames
     cnames = vcat(ta1.colnames, ta2.colnames)
     # check if kwarg to over-ride simple vcat and then if colnames is valid length
-    size(col_names,1) == 1 ? cnames = cnames :       # kwarg not supplied
-    size(col_names,1) == size(vals,2) ? cnames = col_names : error("col_names supplied is not correct size")
+    length(col_names) == size(vals,2) ? cnames = col_names :
```

Comment:
Why change `size(col_names,1)` to `length(col_names)`? I'd prefer to not use `length`